Automattic/woocommerce-payments
# Exclude estimated deposits from useDeposits data hook by default

Fixes #7604 

#### Changes proposed in this Pull Request

This PR re-introduces changes made in #7605 – filtering out by `estimated` deposits from the **Payments → Deposits** screen by excluding the `estimated` deposits from the `useDeposits` data hook by default.

#7605 was reverted to remove it from the WooPayments 6.8 release. This revert is being reverted so it can now be merged and shipped in WooPayments 6.9.

Code changes in this PR should be identical to those in #7605, which were previously approved.

#### Testing instructions

* Ensure your store has an estimated deposit by purchasing a product and setting the deposit schedule to daily, weekly or monthly.
* Visit the **Payments → Deposits** screen.
* Ensure no `Estimated` deposits appear in the list.
* Ensure the summary at the bottom of the table is correct.
